### PR TITLE
Add analog-minimalist

### PR DESCRIPTION
**Pull request by xattsec**
I've made this Minimalist watchface, it was based on "analog-aviator" from eLtMosen.
The "xatt" at the top is not for branding, I just thought it looked cool.

### CODE REVIEW

File: analog-minimalist/usr/share/asteroid-launcher/watchfaces-img/analog-minimalist-hour-ambient.svg [hunk 1]
```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+   width="800"
+   height="800"
+   viewBox="0 0 211.667 211.667"
+   version="1.1"
+   id="svg756"
+   sodipodi:docname="analog-minimalist-hour-ambient.svg"
```
Comment (beroset) [inline analog-minimalist/usr/share/asteroid-launcher/watchfaces-img/analog-minimalist-hour-ambient.svg:8]: None of this inkscape stuff is relevant to the watch.  You should run these SVG files through `svgo` or export as "Optimized SVG" from inkscape to strip out irrelevant metadata.

File: analog-minimalist/usr/share/asteroid-launcher/watchfaces-img/asteroid-logo-black.svg [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+     id="path1097" /><image
+     width="470.59811"
+     height="174.20599"
+     preserveAspectRatio="none"
+     xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAYUAAACQCAYAAADnT3fUAAAAAXNSR0IArs4c6QAAH3pJREFUeF7t
+XQncvtWYvi4zNHZmhkKbFiQRlbShyV6WiJAYMqFkLGEsxURNY0uGQrINRbYi2RKSfS2hZCLLGPsY
+yxjbNb/r/3++z/f/+t73Oc9ynuV97vv3e3//+M65z32uc95zvWe575sICQQCgUAgEAgECgQYSAQC
+gUAgEAgEAksIBCnEXAgEAoFAIBBYRiBIISZDIBAIBAKBQJBCzIFAIBAIBAKBKyMQO4WYFYFAIBAI
+BAKxU4g50B4CknYB8Nn2NK7T9FKST2pZZ6gLBAKBEgTW3ClI2hzAo0aI3iUk31LFbkm7A7hblTot
+l/0RyZNa1tmpOkm2/3EtN/oTADcm+fuW9Sark7QnAH9CukHgZJK/WGoq8O8G9BWtrMN/Fin4i/Dx
```
Comment (beroset) [inline analog-minimalist/usr/share/asteroid-launcher/watchfaces-img/asteroid-logo-black.svg:106]: More than half the file size is to show "xatt" which doesn't really add much to the design, except perhaps for the author.  I'd recommend omitting it.

File: analog-minimalist/usr/share/asteroid-launcher/watchfaces/analog-minimalist.qml [hunk 3]
```diff
@@ -0,0 +1,273 @@
+/*
+ * Copyright (C) 2021 - Timo Könnecke <github.com/eLtMosen>
```
Comment (beroset) [inline analog-minimalist/usr/share/asteroid-launcher/watchfaces/analog-minimalist.qml:2]: If you've made changes to this file, add your own copyright line above this.

**Review by beroset — CHANGES_REQUESTED** [review]
Nice first contribution!   It's missing a modification to the README.md file, which should also mention the licensing of the font used and show two preview image as all the other watchfaces have.

File: analog-minimalist/usr/share/asteroid-launcher/watchfaces/analog-minimalist.qml [hunk 4]
```diff
@@ -0,0 +1,273 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import QtQuick 2.9
+import QtGraphicalEffects 1.15
+
+Item {
+
```
Comment (FlorentRevest) [inline analog-minimalist/usr/share/asteroid-launcher/watchfaces/analog-minimalist.qml:24]: Unnecessary blank line

**Review by FlorentRevest — CHANGES_REQUESTED** [review]
+1 to everything beroset said, happy to merge this afterward though

**Pull request closed without merging**